### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by winterallen** [rancher/rancher#39653]
**Rancher Server Setup**
- Rancher version:
- Installation option (Docker install/Helm Chart):Helm Chart
   - If Helm Chart, Kubernetes Cluster and version (RKE1, RKE2, k3s, EKS, etc):RKE2
- Proxy/Cert Details:

**Information about the Cluster**
- Kubernetes version:
- Cluster Type (Local/Downstream):Downstream
   - If downstream, what type of cluster? (Custom/Imported or specify provider for Hosted/Infrastructure Provider):Imported RKE2

**User Information**
- What is the role of the user logged in? (Admin/Cluster Owner/Cluster Member/Project Owner/Project Member/Custom)
  - If custom, define the set of permissions:Admin

**Describe the bug**

**Downstream cluster rotation certificate updated kubeconfig, rancher operation downstream cluster appears**
![image](https://user-images.githubusercontent.com/17891799/202841098-a33cb7aa-0a7f-4a32-b8c0-0203891a8143.png)

the downstream cluster is REK2 created in advance, and the cluster is imported through rancher cluster management

what do I need to do to get rancher back in control of the downstream cluster

**Comment by winterallen** [comment]
I found a reason:
`/etc/cni/net.d/calico-kubeconfig` 
is not updated after RKE2 cluster certificate rotation

**Comment by bennysp** [comment]
@winterallen Having the same problem. Did you find a workaround?

**Comment by winterallen** [comment]
> @winterallen Having the same problem. Did you find a workaround?

Yes, I successfully updated "/etc/cni/net.d/calico-kubeconfig" by upgrading the downstream RKE2 cluster version to v1.24.7+rke2r1, and the cluster returned to normal, but RancherUI prompts "Failed to communicate with API server during namespace check: Unauthorized", you can try to just upgrade the calico version in the test environment, and observe whether "/etc/cni/net.d/calico-kubeconfig" is updated

**Comment by winterallen** [comment]
> > @winterallen Having the same problem. Did you find a workaround?
> 
> Yes, I successfully updated "/etc/cni/net.d/calico-kubeconfig" by upgrading the downstream RKE2 cluster version to v1.24.7+rke2r1, and the cluster returned to normal, but RancherUI prompts "Failed to communicate with API server during namespace check: Unauthorized", you can try to just upgrade the calico version in the test environment, and observe whether "/etc/cni/net.d/calico-kubeconfig" is updated

**RKE2 version v1.21.5+rke2r2 when not upgraded**
